tracer/time_windows.go

```diff
@@ -0,0 +1,31 @@
+package tracer
+
+import (
+	"golang.org/x/sys/windows"
+	"time"
+)
+
+// This method is more precise than the go1.8 time.Now on Windows
+// See https://msdn.microsoft.com/en-us/library/windows/desktop/hh706895(v=vs.85).aspx
+// It is however ~10x slower and requires Windows 8+.
```

Comment:
The problem is that indeed testing time related stuff is tricky. Since the time.Sleep() function uses the same clocks as the time.Now() timer, any use of time.Sleep() will result in a non 0 difference between two calls to time.Now(). I chose to poll the value of the high precision timer until it changes, and assert that the low precision one didn't change in the meantime.